specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-04-01/managedClusters.json

```diff
@@ -2666,9 +2666,14 @@
           "$ref": "#/definitions/AgentPoolMode"
         },
         "orchestratorVersion": {
+          "type": "string",
+          "title": "The version of Kubernetes specified by the user.",
+          "description": "Both patch version <major.minor.patch> and <major.minor> are supported. When <major.minor> is specified, the latest supported patch version is chosen automatically. Updating the agent pool with the same <major.minor> once it has been created will not trigger an upgrade, even if a newer patch version is available. As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
```

Comment:
Might be good to unify on language used with https://github.com/Azure/azure-rest-api-specs/pull/18821/files as well, since they are describing the same feature I believe.

Reply:
So that might look like this:
description becomes: "**Both patch version <major.minor.patch> (e.g. 1.20.13) and <major.minor> (e.g. 1.20) are supported. When <major.minor> is specified, the latest supported patch version is chosen automatically. Updating the cluster with the same <major.minor> once it has been created (e.g. 1.14.x -> 1.14) will not trigger an upgrade, even if a newer patch version is available.**  As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."

Reply:
should be latest supported GA patch version